src/components/landing/codeChallenge/CodeChallenge.tsx

```diff
@@ -0,0 +1,105 @@
+import { motion } from "framer-motion";
+import * as React from "react";
+import { DISCORD } from "../../../constants/links";
+import Lottie from "react-lottie-player";
+import lottieJson from "../assets/lottie.json";
+import Video from "./assets/code.mp4";
+import Subheader from "../intro/Subheader";
+import GradientButton from "../intro/GradientButton";
+import Header from "../intro/Header";
+import { useNavigate } from "react-router-dom";
+
+const CodeChallenge: React.FC<{}> = (props) => {
+  const navigate = useNavigate();
+  return (
+    <div className={"relative bg-neutral-900 h-screen overflow-hidden"}>
```

Comment:
Use a white background rather than the video background. I recommend removing all of the absolute positioning & converting it to a simpler div (with flexbox) with a white background.

Reply:
Also, be sure to reduce vertical spacing. The section below this one uses `p-10` for padding.